client/components/common/CandidateModals/ProvideFeedbackModal/index.jsx

```diff
@@ -0,0 +1,176 @@
+                                    className={feedbackInputDisabled}
+                                />
+                                <FormCheck.Label
+                                    className={feedbackInputDisabled}
+                                >
+                                    <div>
+                                        <FontAwesomeIcon
+                                            icon={faCommentAlt}
+                                            color="#275CAA"
+                                        />{' '}
+                                        Provide Feedback
+                                    </div>
+                                </FormCheck.Label>
+                                <Form.Text
+                                    className={`radio-text ${feedbackInputDisabled}`}
+                                >
+                                    Provide feedback without explicit approval
+                                </Form.Text>
+                            </FormCheck>
+                            <FormCheck>
+                                <FormCheck.Input
+                                    id="changes-input"
+                                    name="radio-feedback"
+                                    type="radio"
+                                    onClick={radioChange}
+                                    className={changesInputDisabled}
+                                />
+                                <FormCheck.Label
+                                    className={changesInputDisabled}
+                                >
+                                    <div>
+                                        <FontAwesomeIcon
+                                            icon={faTimes}
+                                            color="red"
+                                        />{' '}
+                                        Request Changes
+                                    </div>
+                                </FormCheck.Label>
+                                <Form.Text
+                                    className={`radio-text ${changesInputDisabled}`}
+                                >
+                                    Request Changes that must be addressed
+                                    before approving
+                                </Form.Text>
+                            </FormCheck>
+                        </Form.Group>
+                        <Form.Group>
+                            <Form.Control
+                                className="feedback-text"
+                                as="textarea"
+                                rows={5}
+                            />
+                        </Form.Group>
+                    </Form>
+                </div>
+            }
+            dialogClassName="feedback"
+            handleAction={handleAction}
+            title={
+                <div className="feedback-title">
```

Comment:
VO describes this modal header as having 10 items and stops. The text isn't read. I haven't tested on NVDA/JAWS as yet but this may be worth looking into.